README.md

```diff
@@ -87,7 +87,7 @@ android {
 ```dart
```

Comment:
I have an issue with `get packages` where flutter was grabbing an old version of our plugin.
I had to run `flutter packages upgrade` to upgrade to latest. I think this would only be an issue for updating an existing project but not 100%. 
Might be worth adding `flutter packages upgrade` as a step after `flutter packages get` in the readme to make sure users don't fall into the same trap.

Reply:
@amit-nayar the `flutter packages upgrade` was required when you created a new Flutter project?

Reply:
No, which is why I'm not sure it's necessary in the readme. I thought it might help though for users who may be upgrading their existing projects. But honestly not sure in this case. It should probably be common knowledge for Flutter users. WDYT?

Reply:
I did not have to run the command when I created a new project to test this fix. So I would avoid adding an extra integration step if we can avoid it.